styleguide/source/assets/js/alert.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Even though we haven't good about it we should wrap this inside a drupal function:
```
(function ($, Drupal) {
  Drupal.behaviors.alert = {

//Something goes here

})(jQuery, Drupal);
```